From 8339ccc47af05298612b7f01b8609f9081e9b081 Mon Sep 17 00:00:00 2001
From: Zoltan HERPAI <wigyori@uid0.hu>
Date: Sat, 31 May 2025 13:59:30 +0000
Subject: [PATCH 30/55] riscv: dts: starfive: vf2: add reserved-memory for E24

E24 is an rv32 RISC-V core in the JH7110 package. As the driver is now
available for it, add reserved memory as required.

Signed-off-by: Zoltan HERPAI <wigyori@uid0.hu>
---
 .../dts/starfive/jh7110-starfive-visionfive-2.dtsi     | 10 ++++++++++
 1 file changed, 10 insertions(+)

--- a/arch/riscv/boot/dts/starfive/jh7110-starfive-visionfive-2.dtsi
+++ b/arch/riscv/boot/dts/starfive/jh7110-starfive-visionfive-2.dtsi
@@ -11,6 +11,16 @@
 	aliases {
 		ethernet1 = &gmac1;
 	};
+
+	reserved-memory {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		e24_mem: e24@c0000000 {
+			reg = <0x0 0x6ce00000 0x0 0x1600000>;
+		};
+	};
 };
 
 &gmac1 {
